my-sms/db/migrate/20210208041633_create_titles.rb

```diff
@@ -0,0 +1,34 @@
+class CreateTitles < ActiveRecord::Migration
+  HONORIFICS = %w[Mr Mrs Miss Ms Master]
+
+  def report_student_titles
+    invalid_titles = Student.where("title NOT IN (?)", HONORIFICS)
+    invalid_titles.each do |title|
+    logger.warn "Unexpected Student title #{s.title} in "\
+         "Student id #{s.id}. Migration expects titles "\
+         "in Students column to be one of #{HONORIFICS}."
```

Comment:
Since we're no longer doing any data migrations here, perhaps we can move this method to the rake task

Reply:
The rake task just adds any existing titles to the new column. This is a warning for if the expected set of titles in `HONORIFICS` is significantly different to the ones in the old title column. Its not doing much anymore, so I can see why you would suggest removing it. I had just left it in because it did _something_ and was already there.

Reply:
Yea I get your thinking, if we move it to the rake task it should be OK still, you could even have it as a separate task, something like `report_invalid_titles`

Reply:
Actually, if we keep the `HONORIFICS` var in this file perhaps just leave this bit of code here